gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/medium_tests/gnomad_genomes_GRCh37_chrX_head2500_run_vep.json

```diff
@@ -0,0 +1,85 @@
+[
+  {
```

Comment:
I ended up removing these lines because they were causing errors. According to SO, [JSON doesn't support comments](https://stackoverflow.com/a/244858/4905403). Should I make a note of this TODO elsewhere?

Reply:
I see, okay this is fine for now. Let's remember to refactor these two tests in near future.

Reply:
I've made a note of it on my TODO list.